docs/user/addons.rst

```diff
@@ -46,8 +46,323 @@ Individual configuration options for each addon are available in :guilabel:`Sett
 Addons data and customization
 -----------------------------
 
-If you'd like to do a custom integration with the data used to render Addons,
-you can learn more about this in our :ref:`flyout-menu:custom event integration` docs.
+Addons can be customized using CSS variables and the data used by Addons can be accessed using a custom event.
+
+CSS Variable Customization
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Addons use CSS custom properties (`CSS variables <https://developer.mozilla.org/en-US/docs/Web/CSS/--*>`_) to allow for easy customization.
+To customize addons, add CSS variable definitions to your theme's CSS:
+
+.. code-block:: css
+
+    :root {
+        /* Reduce Read the Docs' flyout font a little bit */
+        --readthedocs-flyout-font-size: 0.7rem;
+
+        /* Reduce Read the Docs' notification font a little bit */
+        --readthedocs-notification-font-size: 0.8rem;
+
+        /* This customization is not yet perfect because we can't change the `line-height` yet. */
+        /* See https://github.com/readthedocs/addons/issues/197 */
+        --readthedocs-search-font-size: 0.7rem;
+    }
+
+CSS Variables reference
```

Comment:
Should use sentence casing:

```suggestion
CSS variables reference
```

Reply:
I should just leave this open: https://docs.readthedocs.com/dev/latest/style-guide.html#titles